boxoffice/views/admin_report.py

```diff
@@ -49,19 +47,88 @@ def row_handler(row):
     (ItemCollection, {'id': 'ic_id'}, 'item_collection'),
     permission='org_admin')
 def attendees_report(item_collection):
-    headers = ['receipt_no', 'ticket_no', 'ticket_id', 'ticket_type', 'attendee_fullname', 'attendee_email', 'attendee_phone']
+
```

Comment:
Please add a comment here that you've hardcoded for `attendee_details` being a dict that needs flattening, and if there's any other dict in future, this code needs updating.